libnmstate/nm/connection.py

```diff
@@ -61,12 +61,21 @@ class _ConnectionSetting:
     def __init__(self, con_setting=None):
         self._setting = con_setting
 
-    def create(self, con_name, iface_name, iface_type, is_controller):
+    def create(
+        self, con_name, iface_name, nm_iface_type, is_controller, iface_type
+    ):
+        # OVS bridges and interfaces can have duplicated names, this way we
+        # avoid the connection names will be duplicated too.
```

Comment:
The `aviod ... will` seems confusing. How about

```
OVS bridge and interfaces could sharing the same interface name, to distinguish them as NM connection level, instead of using interface name as connection name, we append a prefix
```

Reply:
And instead of changing `_ConnectionSetting.create()`, you should change the caller of it.
The `_ConnectionSetting.create()` already took `con_name` as argument, we should not alter that internally.

The caller `create_new_nm_simple_conn()` could decide which connection name should be used base on `iface` it got. This isolate the `_ConnectionSetting` worry about OVS stuff.

Reply:
Done, thank you!